lib/hw_unique_key/Kconfig

```
@@ -18,7 +18,7 @@ config HW_UNIQUE_KEY_LOAD
 	depends on HAS_HW_NRF_CC310
 	depends on NRF_CC3XX_PLATFORM
 	select FPROTECT
-	imply NRFX_NVMC if !SOC_SERIES_NRF54LX
+	imply NRFX_NVMC if !(CRACEN_HW_PRESENT)
```

no parentheses for a single option
```suggestion
	imply NRFX_NVMC if !CRACEN_HW_PRESENT
```